lib/croods/controller/collection.rb

```diff
@@ -27,9 +27,17 @@ def search_collection(list)
         end
       end
 
+      def sort_by_method(list)
+        if params[:order_by].present?
```

Comment:
Not sure if we should make `order_by` optional when `order` is present.

Reply:
I'm thinking about that, If i was to create a search block, it would be more often to use a order (asc/desc) param than the order_by, considering that in my example I'm already doing a custom search.

Reply:
But the order is independent of the search, i.e. you might search a project by name and then order it by deadline.
Here, you'd need to explicitly search by name and then order by name.

Reply:
Sorry, I meant "filter" instead of search in the whole comment

Reply:
Got it. I think this is a third use case, maybe it would make sense to add it now.
1. Search, filter, sort and paginate via params (like when using a table with filters), the most flexible solution
2. Define a specific order to be calculated internally by the API
This would be 3. Calculate the order internally but use the `order` param, right?
Now that I think about it, usually when I saw that king of method, the asc/desc was fixed according to the business rules, such as order users by the ones with most reviews (always desc)

Reply:
@hrangel made it more flexible (and potentially more dangerous), what do you think?